apps/demo/src/app/cart/components/add-to-cart-notification/effects/add-to-cart-notification.effects.ts

```diff
@@ -1,19 +1,46 @@
 import { Injectable } from '@angular/core';
 import { Actions, Effect, ofType } from '@ngrx/effects';
-import { map } from 'rxjs/operators';
+import { map, tap } from 'rxjs/operators';
+import { Store } from '@ngrx/store';
 
 import { DaffCartActionTypes } from '@daffodil/cart';
 
-import { OpenAddToCartNotification } from '../actions/add-to-cart-notification.actions';
+import { OpenAddToCartNotification, AddToCartNotificationActionTypes, CloseAddToCartNotification } from '../actions/add-to-cart-notification.actions';
+import { DaffModalService, DaffModal } from '@daffodil/design';
+import { AddToCartNotificationComponent } from '../components/add-to-cart-notification/add-to-cart-notification.component';
 
 @Injectable()
 export class AddToCartNotificationEffects {
+  private notification: DaffModal;
 
-  constructor(private actions$: Actions) {}
+  constructor(
+    private actions$: Actions,
+    private daffModalService: DaffModalService,
```

Comment:
The DaffModalModule needs to be imported somewhere. The service isn't being provided to demo right now.

Reply:
Fixed in b75505d2cbbe48d71a604ee3a1eb1a5c597ebc24